modules/node-scripts.config.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+const npmscriptsConfig = require('./npmscripts.config');
+
+const {imports} = npmscriptsConfig.build;
+
+module.exports = {
```

Comment:
Can we provide a short comment about how you actually found these symbols? Thinking for when needing to add another dependency

Reply:
My concept of a "short comment":

![image](https://github.com/liferay-frontend/liferay-portal/assets/3273630/691d94a6-475f-42ac-b566-634f23b165f3)

:shrug:

Reply:
done